Audit log token events (#8727)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/audit-log.md

| Event Name                 | Event Type                     | Description                                            |
| -------------------------- | ------------------------------ | ------------------------------------------------------ |

> | Access Token Issued        | access_token.issued            | dbt issued an access token following OAuth authentication for VS Code extension or MCP server. | 

Reviewer: ```suggestion
| Access Token Issued        | access_token.issued            | dbt issued an access token after OAuth sign-in for VS Code extension or Model Context Protocol (MCP) server. | 
```

Reviewer: Hiya Mirna

Following the style guide, I have amended MCP to include the full name and then put MCP in parentheses as it MCP in the access token section was the first place it was mentioned

Reviewer: Hiya Mirna, This is more of a stylistic thing - the other rows in the tables on the page are pretty short so maybe shorten them. This is also a non-blocker / suggestion

> | Access Token Issued        | access_token.issued            | An access token was issued for OAuth authentication (for example, VS Code extension or MCP server) |

Author: ```suggestion
| Access Token Issued        | access_token.issued            | dbt issued an access token following OAuth authentication for VS Code extension or MCP server. | 
```

> | Access Token Issued        | access_token.issued            | dbt issued an access token after OAuth sign-in for VS Code extension or Model Context Protocol (MCP) server. | 

Reviewer 2: This works for now, but in the future we will support oauth for more than just VSCode extension and our mcp server

Author: ah ok maybe i'll just add a 'for exmaple, vs code and mcp....) etc

> | Access Token Issued        | access_token.issued            | dbt issued an access token after OAuth sign-in for VS Code extension or Model Context Protocol (MCP) server. | 

Author: ```suggestion
| Access Token Issued        | access_token.issued            | dbt issued an access token after OAuth sign-in (for example, VS Code extension or Model Context Protocol (MCP) server). | 
```

| Auth Provider Changed      | auth_provider.changed          | Authentication provider settings changed               |
| Credential Login Succeeded | login.password.succeeded       | User successfully logged in with username and password |

> | Refresh Token Issued       | refresh_token.issued           | dbt issued an access token following OAuth authentication for VS Code extension or MCP server.  |

Reviewer: ```suggestion
| Refresh Token Issued       | refresh_token.issued           | dbt issued a refresh token following OAuth authentication for VS Code extension or MCP server.  |
```

Reviewer: Hiya Mirna

This is a non-blocker, I had a look at the screenshot in this [thread](https://dbt-labs.slack.com/archives/C02N953LRDF/p1772727516235649) from KB and the refresh token issue advised "a refresh token was issued..."

> | Refresh Token Issued       | refresh_token.issued           | dbt issued an access token following OAuth authentication for VS Code extension or MCP server.  |

Reviewer: ```suggestion
| Refresh Token Issued       | refresh_token.issued           | dbt issued an access token after OAuth sign-in for VS Code extension or MCP server.  |
```

Reviewer: Hiya Mirna, This is more of a stylistic thing - the other rows in the tables on the page are pretty short so maybe shorten them. This is also a non-blocker / suggestion

> | Refresh Token Issued       | refresh_token.issued           | dbt issued an access token after OAuth sign-in for VS Code extension or MCP server.  |

Reviewer: Hiya @mirnawong1 

This still shows as "...an access token". KB's event log shows the description for a refresh token event is "An access token was issued"

> | Refresh Token Issued       | refresh_token.issued           | dbt issued an access token after OAuth sign-in for VS Code extension or MCP server.  |

Reviewer: ```suggestion
| Refresh Token Issued       | refresh_token.issued           | dbt issued a refresh token after OAuth sign-in for VS Code extension or MCP server.  |
```

> | Refresh Token Issued       | refresh_token.issued           | dbt issued a refresh token after OAuth sign-in for VS Code extension or MCP server.  |

Author: ```suggestion
| Refresh Token Issued       | refresh_token.issued           | dbt issued a refresh token after OAuth sign-in (for example, VS Code extension or MCP server). |
```

> | Refresh Token Issued       | refresh_token.issued           | A refresh token was issued for OAuth authentication (for example, VS Code extension or MCP server) |

Author: ```suggestion
| Refresh Token Issued       | refresh_token.issued           | dbt issued an access token following OAuth authentication for VS Code extension or MCP server.  |
```
